src/app.ts

```diff
@@ -36,6 +37,7 @@ app.enable("trust proxy");
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
+app.use(cookieMiddleware);
```

Comment:
Looking at cookieMiddleware it basically loads the chSession. I think it would be more descriptive to name it chSessionLoader instead of cookieMiddleware.

Reply:
or set it back to sessionMiddleware and call the new middleware 'ptfSessionLoader'

Reply:
Actually, we don't need to rename sessionMiddleware. Keep it as it was and just add the new middleware called something like ptfSessionLoader

Reply:
I have come up with something else see app ts. I thinks we should have some naming symmetry between the two middlewares.

Reply:
Put original name of cookie middleware back and descriptively named ptf session middleware.